### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by Kataderon**

### CODE REVIEW

**Comment by Kataderon** [comment]
https://rpr.cis.luxoft.com/view/RPR%20Plugins/job/RadeonProRenderMayaPluginManual/2135/

File: jobs/Scripts/simpleRender.py [hunk 1]
```diff
@@ -143,7 +143,7 @@ def main(args):
 	except:
 		pass
 
-	maya_scenes = set(re.findall(r'\w*\.ma', test_cases))
+	maya_scenes = set(re.findall(r'scene.*"(\w*\.ma)', test_cases))
```
Comment (MagisterDemens) [inline jobs/Scripts/simpleRender.py]: `scenes = {x.get('scene', '') for x in a}`

File: jobs/Scripts/base_functions.py [hunk 2]
```diff
@@ -20,7 +20,12 @@
 LOGS_DIR = path.join(WORK_DIR, 'render_tool_logs')
 
 
+def logging(massage):
+	print('base_function.py [' + datetime.datetime.now().strftime('%H:%M:%S') + '] ' + massage)
```
Comment (MagisterDemens) [inline jobs/Scripts/base_functions.py]:
```suggestion
	print(' >>> [RPR TEST] ' + '[' + datetime.datetime.now().strftime('%H:%M:%S') + '] ' + message)
```

**Comment by Kataderon** [comment]
https://rpr.cis.luxoft.com/view/RPR%20Plugins/job/RadeonProRenderMayaPluginManual/2154/

File: jobs/Scripts/simpleRender.py [hunk 3]
```diff
@@ -142,7 +143,7 @@ def main(args):
 	except:
 		pass
 
-	maya_scenes = {x.get('scene', '') for x in a if not x}
+	maya_scenes = {x.get('scene', '') for x in test_cases if not x}
```
Comment (MagisterDemens) [inline jobs/Scripts/simpleRender.py]: fix

**Pull request merged**